src/utils/api/orderService/orderService.ts

```diff
@@ -0,0 +1,48 @@
+export interface OrderedMeal {
+    /** Meal's id */
+    id: string;
```

Comment:
```suggestion
    /** 
     * Meal's id
     */
    id: string;
```
jsdoc'и пишутся минимум в три строчки

Reply:
выполнено